packages/web-twig/src/Resources/components/Toast/__tests__/__fixtures__/toastBarLink.twig

```diff
@@ -4,7 +4,6 @@
 <ToastBarLink
     href="https://www.google.com"
     target="_blank"
-    isUnderlined={false}
     isDisabled={false}
```

Comment:
why not underlined="never"?

Reply:
we should test it somehow

Reply:
and does it work? Shouldn't the snapshot be updated? 😨

Reply:
i don't know why it was there, there is no functionality for this prop in this component.
this component is hardcoded to be always underlined in the code.

Reply:
ToastBarLink doesn't have this prop in readme. 
The only way how you can change the underlining of a link inside of a toast in twig is to use dynamic toast -> use JS plugin and Toast class and let javascript generate the structure of toast.

Reply:
ok, it is a bit weird, but it is twig... :D

Reply:
its the same in react 😁